ampath-simple-app/src/app/app.component.html

```diff
@@ -1,5 +1,4 @@
 <!--The content below is only a placeholder and can be replaced.-->
-<mat-toolbar>
-  <span>My Application</span>
-</mat-toolbar>
+<h1>{{ title }}</h1>
+<app-tabs></app-tabs>
```

Comment:
@mrogers4 Is your tab component the replacement for the toolbar? I just want to check before I merge this.

Reply:
Yes, that's correct!